runtime/compiler/il/J9DataTypes.cpp

```diff
@@ -1057,7 +1057,7 @@ J9::DataType::getBCDPrecisionFromString(char *str, TR::DataType dt) // +123 prec
       }
    else
       {
-      TR_ASSERT(isdigit(str[0]),"expecting a minus/plus sign or a decimal digit and not 0x%x\n",str[0]);
+      TR_ASSERT(OMR_ISDIGIT(str[0]),"expecting a minus/plus sign or a decimal digit and not 0x%x\n",str[0]);
```

Comment:
While you're here, please add the missing space after comma; here and on line 1076.